PERSONAS.md

```diff
@@ -0,0 +1,27 @@
+## What is a persona?
+
+Personas are fictional characters, which you create based upon your research in order to represent the different user types that might use your service, product, site, or brand in a similar way. Creating personas will help you to understand your users’ needs, experiences, behaviours and goals.
+
+[_Source_](https://www.interaction-design.org/literature/article/personas-why-and-how-you-should-use-them)
+
+## What are the main personas for Sing for Needs?
+
+We identified 3 main groups for this mission: the artists, the causes (fundraisers) and the donors
+
+## This doesn't like like a typical persona, why is that?
```

Comment:
maybe you meant to say `this doesn't sound like a typical persona`? instead of the two likes?

Reply:
Yep, that's what I wanted!